server/plugin/command/create_event.go

```diff
@@ -12,10 +13,15 @@ import (
 
 func getCreateEventFlagSet() *flag.FlagSet {
 	flagSet := flag.NewFlagSet("create", flag.ContinueOnError)
-	flagSet.String("subject", "", "Subject of the Event (no spaces for now)")
+	flagSet.Bool("help", false, "show help")
```

Comment:
0/5 we should use the same style consistently everywhere, I believe other plugins usually use the `...Var` form of the functions.

Reply:
@levb, can you add some context to this comment?  Are you suggesting some sort of variadic function usage?

Reply:
No, see `flagSet.StringVar` etc. It's just a slightly different way of using the FlagSet. Not even a better one, just advocating that we do the same everywhere.

Reply:
I did a quick search through the plugins I have cloned it looks like most use the non-`Var` form of the functions.  Specifically, I was looking at the cloud plugin as an example. 

Perhaps we need to actually we can discuss this in our meeting?  I assume there will be arguments for both methods.

Reply:
Leave as is, it's fine.